lib/seccomp/CSystemCallFilter_Windows.cc

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+#include <core/CWindowsError.h>
+
+#include <WinBase.h>
+
+
+namespace ml {
+namespace seccomp {
+
+CSystemCallFilter::CSystemCallFilter() {
+    HANDLE job = CreateJobObject(nullptr, nullptr);
+    if (job == nullptr) {
+        LOG_ERROR(<< "Failed to create Job Object: " << ml::core::CWindowsError().errorString());
+    }
+
+    JOBOBJECT_BASIC_LIMIT_INFORMATION limits;
+
+    if (QueryInformationJobObject(job, JobObjectBasicLimitInformation,
+                                  &limits, sizeof(limits), nullptr) == 0) {
+        LOG_ERROR(<< "Error querying Job Object information: " << ml::core::CWindowsError().errorString());
```

Comment:
You can just use `<< ml::core::CWindowsError()`.  There's no need to call `errorString()`.

Same in the other places it's used.